### PR TITLE
Refactor integration

### PR DESCRIPTION
**Pull request by bmxam**

### CODE REVIEW

File: test/integration/test_integration.jl [hunk 1]
```diff
@@ -182,49 +226,92 @@ end
...
+            _c2f = c2f[icell]
+            kface = _c2f[1]
+            _f2n = f2n[kface]
+            finfo_1 = FaceInfo(c, c, faces(mesh)[kface], get_nodes(mesh, _f2n), _f2n, kface)
+            kface = _c2f[2]
+            _f2n = f2n[kface]
+            finfo_2 = FaceInfo(c, c, faces(mesh)[kface], get_nodes(mesh, _f2n), _f2n, kface)
+            kface = _c2f[3]
+            _f2n = f2n[kface]
+            finfo_3 = FaceInfo(c, c, faces(mesh)[kface], get_nodes(mesh, _f2n), _f2n, kface)
```
Comment (ghislainb) [inline test/integration/test_integration.jl:311]: Can you replace with the following if it works:
```julia
finfo_1, finfo_2, finfo_3 = map( Base.Fix1(FaceInfo, mesh), c2f[icell])
```

File: test/integration/test_integration.jl [hunk 2]
```diff
@@ -182,49 +226,92 @@ end
...
+                finfo_2 =
+                    FaceInfo(c, c, faces(mesh)[kface], get_nodes(mesh, _f2n), _f2n, kface)
+                kface = _c2f[3]
+                _f2n = f2n[kface]
+                finfo_3 =
+                    FaceInfo(c, c, faces(mesh)[kface], get_nodes(mesh, _f2n), _f2n, kface)
+                kface = _c2f[4]
+                _f2n = f2n[kface]
+                finfo_4 =
+                    FaceInfo(c, c, faces(mesh)[kface], get_nodes(mesh, _f2n), _f2n, kface)
```
Comment (ghislainb) [inline test/integration/test_integration.jl:292]: Can you replace with the following if it works:
```julia
finfo_1, finfo_2, finfo_3, finfo_4 = map( Base.Fix1(FaceInfo, mesh), c2f[icell])
```

Comment (ghislainb) [reply test/integration/test_integration.jl:292]: (and remove the two previous lines)

Comment (bmxam) [reply test/integration/test_integration.jl]: yes I can

**Review by ghislainb — APPROVED** [review]

**Pull request merged**